src/lamplib/src/genny/tasks/documentation_generator.py

```diff
@@ -71,12 +73,18 @@ def _get_workload_files(self, roots: list[str]) -> list[str]:
                         yaml_files.append(path.join(dirpath, filename))
         return yaml_files
 
-    def _get_workload_from_file(self, yaml_path: str) -> Workload:
+    def _get_workload_from_file(self, yaml_path: str, documentation_type: str) -> Workload:
         with open(yaml_path, "r") as f:
             workload_yaml = yaml.safe_load(f)
             workload_name = PurePosixPath(yaml_path).stem
+
             path = yaml_path.replace(self.genny_repo_root, "")
             team = self.mothra_service.get_team(workload_yaml.get("Owner"))
+            if documentation_type == "workload":
+                workload_name_snake_case = self._workload_camel_to_snake(workload_name)
+                task_page = f"https://evergreen.mongodb.com/task_history/sys-perf/{workload_name_snake_case}"
```

Comment:
Users may use the metadata from a project other than sys-perf (i.e. sys-perf-8.0) to reach this page, but we always just link to sys-perf as we are unable to know which project the user is linking in from.